src/loaders/omeXML.js

```diff
@@ -21,8 +21,17 @@ export default class OMEXML {
     this.SizeY = Number.parseInt(Pixels['@_SizeY']);
     this.DimensionOrder = Pixels['@_DimensionOrder'];
     this.Type = Pixels['@_Type'];
-    this.PhysicalSizeYUnit = Pixels['@_PhysicalSizeYUnit'];
-    this.PhysicalSizeXUnit = Pixels['@_PhysicalSizeXUnit'];
+    const PhysicalSizeYUnit = Pixels['@_PhysicalSizeYUnit'];
+    // This µ character is not well handled - I got odd behavior but this solves it.
+    this.PhysicalSizeYUnit =
+      PhysicalSizeYUnit && PhysicalSizeYUnit.includes('µm')
+        ? 'µm'
+        : this.PhysicalSizeXUnit;
+    const PhysicalSizeXUnit = Pixels['@_PhysicalSizeXUnit'];
+    this.PhysicalSizeXUnit =
+      PhysicalSizeXUnit && PhysicalSizeXUnit.includes('µm')
+        ? 'µm'
+        : this.PhysicalSizeXUnit;
```

Comment:
is this field often missing?

Reply:
Can you describe what the 'odd behavior' is?

Reply:
I'll leave a clearer note.

Reply:
There was a prepended character.  Could have been an error but the µm is kind of wonky so it could be a python-javascript difference.